lib/system/trace.toit

```diff
@@ -18,8 +18,13 @@ send_trace_message message/ByteArray -> none:
   handled := false
   try:
     service := service_
-    if not service: service = service_ = (TraceServiceClient --no-open).open
-    handled = service.handle_trace message
```

Comment:
The common case is actually that `service` is null here, so I'd rather not get another exception thrown in that case.